crates/biome_js_analyze/tests/specs/nursery/useCollapsedIf/invalid.js

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
Could you check the original test file and add cases to increase coverage?
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/test/no-lonely-if.mjs